test/ae_database/databasesetup.php

```diff
@@ -0,0 +1,254 @@
+<?php
+include "AEData.inc";
+
+sqlsrv_configure( 'WarningsReturnAsErrors', 1 );
+sqlsrv_configure( 'LogSeverity', SQLSRV_LOG_SEVERITY_ALL );  
+
+$databaseName = 'AEDemo';
+$uid = 'yourUsername';
+$pwd = 'yourPassword';
+$server = 'yourServer';
+
+$connectionInfo = array( "Database"=>$databaseName, "UID"=>$uid, "PWD"=>$pwd);
+
```

Comment:
I'm not going to change this for now since I don't really know how the directory structure is going to look like at the end. Maybe these tests will be put into the functional test folder and they can all just use MSSetup.inc.